tests/models/test_lora_layers.py

```diff
@@ -329,16 +319,10 @@ def test_text_encoder_lora_remove_monkey_patch(self):
         outputs_without_lora = pipe.text_encoder(**dummy_tokens)[0]
         assert outputs_without_lora.shape == (1, 77, 32)
 
-        # create lora_attn_procs with randn up.weights
-        text_attn_procs = create_text_encoder_lora_attn_procs(pipe.text_encoder)
-        set_lora_up_weights(text_attn_procs, randn_weight=True)
-
         # monkey patch
-        pipe._modify_text_encoder(text_attn_procs)
+        params = pipe._modify_text_encoder(pipe.text_encoder, pipe.lora_scale)
 
-        # verify that it's okay to release the text_attn_procs which holds the LoRAAttnProcessor.
-        del text_attn_procs
-        gc.collect()
+        set_lora_weights(params, randn_weight=True)
```

Comment:
Because we no-longer pass our own modules in to the monkey patching function, we don't have to test that it's ok to delete them

Reply:
(testing that it's ok to delete something in python should generally raise a red flag)